securedrop/journalist_app/admin.py

```diff
@@ -55,18 +62,20 @@ def manage_config():
 
     @view.route('/update-submission-preferences', methods=['POST'])
     @admin_required
-    def update_submission_preferences():
+    def update_submission_preferences() -> Optional[werkzeug.Response]:
         form = SubmissionPreferencesForm()
         if form.validate_on_submit():
             # The UI prompt ("prevent") is the opposite of the setting ("allow"):
             flash(gettext("Preferences saved."), "submission-preferences-success")
             value = not bool(request.form.get('prevent_document_uploads'))
             InstanceConfig.set('allow_document_uploads', value)
             return redirect(url_for('admin.manage_config'))
+        else:
```

Comment:
It's a nit, but the `else:` is unnecessary. You don't have to change it; we're doing it this way throughout and it should probably be addressed in a separate PR. I just wanted to note it for the person that takes that on.

Reply:
I like the readability of having an explicit `else` but this method is small anyway.